tests/test_kubernetes_tools.py

```diff
@@ -2925,7 +2923,12 @@ def test_get_volume_claim_templates(self):
             assert len(ret) == 2
 
     def test_get_storage_class_name_default(self):
-        pv = kubernetes_tools.PersistentVolume()
+        pv = kubernetes_tools.PersistentVolume(
+            mode="rw",
+            size=1000,
+            container_path="/tmp",
+            storage_class_name="ebs",
+        )
```

Comment:
i think this is changing the test behavior - what was the error here?

Reply:
this is why I added these keys